week-3-practise/level-1/3-filter.js

```diff
@@ -545,7 +545,10 @@ var streetNames = [
   "Wrestler's Court"
 ];
 
```

Comment:
Try to branch from master for each new week so stuff from the previous week isn't included. Skipping to the week 4 stuff.